MANIFEST.in

```diff
@@ -1,9 +1,14 @@
 include LICENSE.txt README CHANGES.txt
 include requirements*.txt
+include test.pdf
+include tests.py
 
-include docs/Makefile
-recursive-include docs *.py *.rst
-include docs/_static/.keep_dir
-prune docs/_build
+graft dokang_pdf
 
-recursive-include *.py
\ No newline at end of file
+prune docs
+prune tests
```

Comment:
There is no "tests" or "docs" folder, I think these two lines can be removed.

However, there are files named "tests.py" and "test.pdf" that we do not want to ship. Lines 3-4 above should hence be replaced by "exclude xxx".